src/ansi-c/scanner.l

```diff
@@ -499,11 +499,7 @@ enable_or_disable ("enable"|"disable")
 
 <GRAMMAR,GCC_ATTRIBUTE3>{
 "auto"          { loc(); return TOK_AUTO; }
-"_Bool"         { if(PARSER.cpp98)
-                    return make_identifier();
-                  else
-                  { loc(); return TOK_BOOL; }
-                }
+"_Bool"         { return conditional_keyword(!PARSER.cpp98, TOK_BOOL); }
```

Comment:
As discussed offline: we really should rename `PARSER.cpp98` to something like `PARSER.cpp98_or_later` as otherwise this is very confusing.

Reply:
Will do as separate PR.